include/fmt/core.h

```diff
@@ -96,6 +96,20 @@
 #  define FMT_CONSTEXPR_DECL
 #endif
 
+#if __cplusplus >= 201402L
+#  define FMT_CONSTEXPR14 constexpr
+#else
+#  define FMT_CONSTEXPR14
+#endif
```

Comment:
`FMT_CONSTEXPR` is only enabled on C++14 so you don't need a separate `FMT_CONSTEXPR14`. Please remove.

Reply:
Done